bokeh/models/widgets/widget.py

```diff
@@ -14,4 +14,4 @@
 class Widget(LayoutDOM):
     """ A base class for all interactive widget types. """
 
-    responsive = Override(default='fixed')
+    sizing_mode = Override(default='fixed')
```

Comment:
This Override isn't necessary anymore - it was at some point in the past, but then everything went to fixed.